bulma-rtl.sass

```diff
@@ -0,0 +1,10 @@
+/*! bulma.io v0.4.0 | MIT License | github.com/jgthms/bulma */
+@charset "utf-8"
+
+$right: left
+$left: right
+
+html
+  direction: rtl
+
+@import "bulma"
```

Comment:
> Components in an RTL page will always be RTL, right? There's no common reason to mix both RTL and LTR styled components, I think.

@mightyiam actually there is a case for LTR components in RTL page.
for example, an email input field should be always LTR, even if the page is RTL.
this can be done by decorating the field specifically with `direction:ltr`.